core/src/components/datetime/datetime.tsx

```diff
@@ -315,6 +315,20 @@ export class Datetime implements ComponentInterface {
     return pickerOptions;
   }
 
+  /**
+   * Gets a date value given a format
+   * Defaults to the current date if
+   * no date given
+   */
+  private getDateValue(date: DatetimeData, format: string): number {
```

Comment:
This could probably go in the external Datetime util file since it doesn't use anything specific to this component

Reply:
ok all set! 👍